src/bin/cratesfyi.rs

```diff
@@ -308,5 +309,6 @@ fn logger_init() {
                 record.args())
     });
     builder.parse(&env::var("RUST_LOG").unwrap_or("cratesfyi=info".to_owned()));
-    builder.init();
+
+    rustwide::logging::init_with(builder.build());
```

Comment:
I'm assuming this initializes the global logger as well? I want to make sure we can still depend on the docs-rs log itself, since this replaced the `init` line.

Reply:
Yeah, rustwide's logging wraps another logger (docs.rs's env_logger in this case): it processes the records and then it forwards them to the wrapped logger.